Assets/Scripts/Save System/ISaveableData.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Cosmobot
+{
+    // all objects that need to be saved should implement this interface
+    public interface ISaveableData
+    {
+        bool LoadData(GameData data);
+        bool SaveData(in GameData data);
```

Comment:
good enough (only because we don't have anything to save now), but maybe it would be better to use some kind of input/output Stream?